srcpkgs/dotnet-runtime/files/LICENSE

```diff
@@ -0,0 +1,23 @@
+The MIT License (MIT)
```

Comment:
Please do not vendor licenses. The current workaround for licenses that are not part of the package is to fetch them via distfiles.